pkg/openapiclient/local_crds.go

```diff
@@ -108,15 +118,14 @@ func (k *localCRDsClient) Paths() (map[string]openapi.GroupVersion, error) {
 			if err != nil {
 				return nil, err
 			}
-
-			gvr := gvk.GroupVersion().WithResource(crd.Spec.Names.Plural)
 			sch.AddExtension("x-kubernetes-group-version-kind", []interface{}{gvkObj})
-
+			// Add schema extension to propagate the scope
+			sch.AddExtension("x-kubectl-validate-scope", strings.ToLower(string(crd.Spec.Scope)))
```

Comment:
Since the value of this is an `enum`, I think it is more correct to leave the CamelCase version as we do in CRDs themselves.

Reply:
done